tfhe/benches/core_crypto/ks_bench.rs

```diff
@@ -402,15 +514,52 @@ mod cuda {
 
 #[cfg(feature = "gpu")]
 use cuda::cuda_keyswitch_group;
+use tfhe::shortint::parameters::{
+    COMP_PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64, PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64,
+};
 
 pub fn keyswitch_group() {
-    let mut criterion: Criterion<_> =
-        (Criterion::default().sample_size(2000)).configure_from_args();
+    let mut criterion: Criterion<_> = (Criterion::default()
+        .sample_size(15)
+        .measurement_time(std::time::Duration::from_secs(60)))
+    .configure_from_args();
     keyswitch(&mut criterion, &benchmark_parameters_64bits());
     keyswitch(&mut criterion, &benchmark_parameters_32bits());
 }
 
+pub fn packing_keyswitch_group() {
+    let mut criterion: Criterion<_> = (Criterion::default()
+        .sample_size(10)
+        .measurement_time(std::time::Duration::from_secs(30)))
+    .configure_from_args();
+    packing_keyswitch(
+        &mut criterion,
+        "packing_keyswitch",
+        &benchmark_compression_parameters(),
+        keyswitch_lwe_ciphertext_list_and_pack_in_glwe_ciphertext,
+    );
+    packing_keyswitch(
+        &mut criterion,
+        "packing_keyswitch",
+        &benchmark_compression_parameters(),
+        keyswitch_lwe_ciphertext_list_and_pack_in_glwe_ciphertext,
+    );
+    packing_keyswitch(
+        &mut criterion,
+        "par_packing_keyswitch",
+        &benchmark_compression_parameters(),
+        par_keyswitch_lwe_ciphertext_list_and_pack_in_glwe_ciphertext,
+    );
+    packing_keyswitch(
+        &mut criterion,
+        "par_packing_keyswitch",
+        &benchmark_compression_parameters(),
+        par_keyswitch_lwe_ciphertext_list_and_pack_in_glwe_ciphertext,
+    );
```

Comment:
wbhy are the functions written twice here ?

Reply:
Leftovers from `u64` and `u32` versions.